### ISSUE TITLE
Error: Could not find gradle.plugin.org.openbakery:plugin:0.15.1

### ISSUE THREAD

**Issue by adil-hussain-84** [openbakery/gradle-xcodePlugin#377]
I just tried updating my `build.gradle` file to get the plugin from `https://plugins.gradle.org/m2/` instead of `http://repository.openbakery.org` by following the guidance [here](https://plugins.gradle.org/plugin/org.openbakery.xcode-plugin) as follows...

```
buildscript {
  repositories {
    maven {
      url "https://plugins.gradle.org/m2/"
    }
  }
  dependencies {
    classpath "gradle.plugin.org.openbakery:plugin:0.15.1"
  }
}
```

... but I'm getting an error message as follows:

> Could not find gradle.plugin.org.openbakery:plugin:0.15.1
> Searched in the following locations:
> https://plugins.gradle.org/m2/gradle/plugin/org/openbakery/plugin/0.15.1/plugin-0.15.1.pom
> https://plugins.gradle.org/m2/gradle/plugin/org/openbakery/plugin/0.15.1/plugin-0.15.1.jar
> ...

If I change `0.15.1` to `0.15.0` then it works. Anyone know what's going on here?

**Comment by paleozogt** [comment]
@adil-hussain-84 I'm seeing this as well.

Using the `id` syntax
```
plugins {
    id "org.openbakery.xcode-plugin" version "0.15.1"
}
```
fails with
```
> Could not find gradle.plugin.org.openbakery:plugin:0.15.1.
 Searched in the following locations:
     https://plugins.gradle.org/m2/gradle/plugin/org/openbakery/plugin/0.15.1/plugin-0.15.1.pom
     https://plugins.gradle.org/m2/gradle/plugin/org/openbakery/plugin/0.15.1/plugin-0.15.1.jar
 Required by:
     project :asm
```
while 
```
plugins {
    id "org.openbakery.xcode-plugin" version "0.15.0"
}
```
works.

**Comment by paleozogt** [comment]
Trying to pull the plugin directly,
```
wget https://plugins.gradle.org/m2/gradle/plugin/org/openbakery/plugin/0.15.1/plugin-0.15.1.pom
```
fails with a `HTTP request sent, awaiting response... 404 Not Found`

while 
```
wget https://plugins.gradle.org/m2/gradle/plugin/org/openbakery/plugin/0.15.0/plugin-0.15.0.pom
```
works and downloads the POM.

Something is busted with the this plugin's deployment.

**Comment by renep** [comment]
There was a but in the com.gradle.plugin-publish plugin. I have updated to the lastest version and published a version 0.15.2 that should work now.

**Comment by adil-hussain-84** [comment]
Thanks for the update @renep. However, now I'm finding that this syntax...

```
buildscript {
  repositories {
    maven {
      url "https://plugins.gradle.org/m2/"
    }
  }
  dependencies {
    classpath "gradle.plugin.org.openbakery:plugin:0.15.1"
  }
}
```

 ... fails.

While this syntax...

```
buildscript {
    repositories {
        maven {
            url('https://plugins.gradle.org/m2/')
        }
    }
}

plugins {
    id 'org.openbakery.xcode-plugin' version '0.15.2'
}
```

... succeeds.

**Comment by renep** [comment]
Yes, I cannot fix the 0.15.1 version, because I cannot delete the version from plugins.gradle.org. This is the reason why I increased the version number.

**Comment by adil-hussain-84** [comment]
Hey @renep, what I meant in my last comment was that the `plugins` syntax works for the 0.15.2 release but the `classpath` syntax does not work for the 0.15.2 release. Perhaps not a big deal. Just making you aware.
